mk/dev.mk

```diff
@@ -46,12 +46,7 @@ TOOLS_DEPS_LOCK_FILE=mk/dependencies/deps.lock
 # it's important that everything lands in $(CI_TOOLS_DIR) to be able to cache this folder in CI and speed up the build.
 .PHONY: dev/tools
 dev/tools: ## Bootstrap: Install all development tools
-	@mkdir -p $(CI_TOOLS_BIN_DIR) $(CI_TOOLS_DIR)/protos
-	# Also compute a hash to use for caching
-	FILES=`find $(TOOLS_DEPS_DIRS) -name '*.sh' | sort`; \
-		for i in $${FILES}; do OS=$(GOOS) ARCH=$(GOARCH) $$i ${CI_TOOLS_DIR}; done; \
```

Comment:
the original issue here is that `make` doesn't break when an iteration fails. so these needed a `|| exit 1`. maybe we need to consistently set `.SHELLFLAGS` to add `-ec`.